test/conformance/testing/include/uur/fixtures.h

```diff
@@ -209,6 +209,43 @@ template <class T> struct urQueueTestWithParam : urContextTestWithParam<T> {
     ur_queue_handle_t queue;
 };
 
+struct urProfilingQueueTest : urContextTest {
+    void SetUp() override {
+        UUR_RETURN_ON_FATAL_FAILURE(urContextTest::SetUp());
+        ur_queue_property_t props[] = {UR_QUEUE_PROPERTIES_FLAGS,
+                                       UR_QUEUE_FLAG_PROFILING_ENABLE, 0};
+        urQueueCreate(this->context, this->device, props, &queue);
```

Comment:
Should there be an EXPECT_SUCCESS in both of the SetUp() methods for creating the queue?

Reply:
Good spot, I think yes.

Reply:
Thanks. Fixed